src/Http/Action/Security/CentralAction.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Security;
+
+use OpenCFP\Domain\Services;
+use Symfony\Component\HttpFoundation;
+use Symfony\Component\Routing;
+
+final class CentralAction
+{
+    /**
+     * @var Services\Authentication
+     */
+    private $authentication;
+
+    /** @var int */
+    private $clientId;
+
+    /** @var string */
+    private $redirectUri;
+
+    /** @var string */
+    private $authorizeUrl;
+
+    /**
+     * @var Routing\Generator\UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    /** @var string */
+    private $sso;
+
+    /**
+     * LoginAction constructor.
+     *
+     * @param Services\Authentication                 $authentication
+     * @param Routing\Generator\UrlGeneratorInterface $urlGenerator
+     * @param int                                     $clientId
+     * @param string                                  $redirectUri
+     * @param string                                  $authorizeUrl
+     * @param string                                  $sso
+     */
+    public function __construct(
+        Services\Authentication $authentication,
+        Routing\Generator\UrlGeneratorInterface $urlGenerator,
+        int $clientId,
+        string $redirectUri,
+        string $authorizeUrl,
+        string $sso
+    ) {
+        $this->authentication = $authentication;
+        $this->urlGenerator   = $urlGenerator;
+        $this->clientId       = $clientId;
+        $this->redirectUri    = $redirectUri;
+        $this->authorizeUrl   = $authorizeUrl;
+        $this->sso            = $sso;
```

Comment:
This field is actually never used, can we remove it, as well as the dependency on the `$sso` parameter?

Reply:
Removed